encoding/thrift/register.go

```diff
@@ -49,6 +49,12 @@ type UnaryHandler func(context.Context, wire.Value) (Response, error)
 // OnewayHandler is a convenience type alias for functions that act as OnewayHandlers.
 type OnewayHandler func(context.Context, wire.Value) error
 
+// UnaryStreamHandler is a convenience type alias for functions that act as StreamHandlers.
+type UnaryStreamHandler func(context.Context, stream.Reader, stream.Writer) (Response, error)
+
+// OnewayStreamHandler is a convenience type alias for functions that act as OnewayStreamHandlers.
+type OnewayStreamHandler func(context.Context, stream.Reader) error
+
```

Comment:
Not sure to follow here, what are those and how are they used? In HandlerSpec I am seeing a `NoWireHandler` and based on discussion in https://github.com/yarpc/yarpc-go/pull/2084, I thought we would not have different handlers for no wire unary and oneway.

Reply:
Good catch, thanks.  This was a result from earlier commits before we had decided on the `NoWireHandler` mechanisms.

Removed.